src/util/UnixSignalHandler.cpp

```diff
@@ -0,0 +1,86 @@
+#include <QtGlobal>
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
+#    include <QtSystemDetection>
```

Comment:
This is included in `<QtGlobal>` ([docs](https://doc.qt.io/qt-6/qtglobal.html))

Reply:
Fixed in [`c91f894` (#6053)](https://github.com/Chatterino/chatterino2/pull/6053/commits/c91f894f6b12b32ce1645925f6df16c1d921c1b2) - Thanks!